pkgs/development/ocaml-modules/re/default.nix

```
@@ -1,16 +1,29 @@
 { lib, fetchurl, buildDunePackage, ocaml, ounit, seq }:
 
+let version_sha = if lib.versionAtLeast ocaml.version "4.08"
+  then
+    {
+      version = "1.10.4";
+      sha256 = "sha256-g+s+QwCqmx3HggdJAQ9DYuqDUkdCEwUk14wgzpnKdHw=";
+    }
+  else
+    {
+      version = "1.9.0";
+      sha256 = "1gas4ky49zgxph3870nffzkr6y41kkpqp4nj38pz1gh49zcf12aj";
+    };
+in
+
 buildDunePackage rec {
   pname = "re";
-  version = "1.9.0";
+  version = version_sha.version;
 
-  minimumOCamlVersion = "4.02";
+  minimalOCamlVersion = "4.02";
```

```suggestion
  minimumOCamlVersion = "4.02";
```
https://github.com/NixOS/nixpkgs/blob/350fd0044447ae8712392c6b212a18bdf2433e71/pkgs/build-support/ocaml/dune.nix#L10 both are supported

---

Please read the documentation before writing such annoying reviews.

---

Sorry, I confused the two about which one we are supposed to use

---

@ulrikstrid Your previous commit was right. The comment by **Et7f3** was misleading.